Formula/opencv.rb

```diff
@@ -35,6 +35,7 @@ class Opencv < Formula
   depends_on "openblas"
   depends_on "openexr"
   depends_on "openjpeg"
+  depends_on "openvino"
```

Comment:
Could you comment on why this is useful to add as a dependency? I'll note that using this is a little worrying, since `openvino` presumably links statically to `protobuf@21`, while `opencv` uses `protobuf`.

Reply:
Forgot to copy description from the original [PR ](https://github.com/Homebrew/homebrew-core/pull/134427)

The goal of this PR is to improve performance of OpenCV DNN module, which has integration with OpenVINO as an optimization backend. Currently, OpenCV DNN module is using self-written not highly optimized kernels, which are not portable between platforms, while OpenVINO uses platform specific backend for x86_64 and arm64 platforms as well as supports Intel / ADM GPUs on Linux.

An article about OpenVINO integration into OpenCV is https://learnopencv.com/using-openvino-with-opencv/
Now, the performance delta is even bigger.

> since openvino presumably links statically to protobuf@21, while opencv uses protobuf

It's not a problem at all. OpenVINO [hides](https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/compile_flags/os_flags.cmake#L487-L505) symbols of its dependencies and Protobuf is not a public dependency.

> It's also not clear to me why we're not doing BUILD_SHARED_LIBS=ON in openvino

`BUILD_SHARED_LIBS=ON` is a default in OpenVINO ([link](https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/features.cmake#L51))